PinpointKit/PinpointKit/Sources/Core/ShakeDetectingWindow.swift

```diff
@@ -29,7 +29,17 @@ open class ShakeDetectingWindow: UIWindow {
     
 	required public init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
-	}
+	}	
+    /**
+     Initializes a `ShakeDetectingWindow`.
+
+     - parameter windowScene:The window scene.
+     - parameter delegate: An object to notify when a shake motion event occurs.
+     */
+    public init(windowScene: UIWindowScene, delegate: ShakeDetectingWindowDelegate) {
```

Comment:
* PinpointKit supports versions of iOS prior to the introduction of `UIWindowScene`. Could you add the appropriate OS check? i.e. `@available(iOS 13.0, *)`. Currently this won’t build for me in the PinpointKitExample project.
* Could you please move this new initializer above the `UIWindow` MARK after the other custom initializer?